package.json

```diff
@@ -14,7 +14,7 @@
     "test:all": "ember try:each"
   },
   "engines": {
-    "node": "6.* || 8.* || >= 10.*"
+    "node": "^4.5 || 6.* || >= 7.*"
```

Comment:
I think what we had here is fine, since Node 4 is deprecated. We should just update Travis to run with Node 6 instead

Reply:
Ok, wasn't sure if we wanted to support users of node 4